phpunit/functional/Glpi/Form/Export/FormSerializerTest.php

```diff
@@ -56,6 +56,35 @@ public static function setUpBeforeClass(): void
         parent::setUpBeforeClass();
     }
 
+    public function testSingleFormExportFileName(): void
+    {
+        // Arrange: create a single form
+        $builder = new FormBuilder("My form");
+        $form = $this->createForm($builder);
+
+        // Act: export form
+        $result = self::$serializer->exportFormsToJson([$form]);
+
+        // Assert: filename should be the slugified form name + .json
+        $this->assertEquals("my-form.json", $result->getFileName());
+    }
+
+    public function testMultipleFormExportFileName(): void
+    {
+        // Arrange: create 7 forms
+        $forms = [];
+        foreach (range(1, 7) as $i) {
+            $builder = new FormBuilder("Form $i");
+            $forms[] = $this->createForm($builder);
+        }
+
+        // Act: export forms
+        $result = self::$serializer->exportFormsToJson($forms);
+
+        // Assert: filename should reference the number of forms
+        $this->assertEquals("export-of-7-forms.json", $result->getFileName());
```

Comment:
When only one form is exported, having no timestamp/hash/random part in the filename is not an issue, but when exporting multiple forms, it can be an issue. For instance, you may want to export 3 forms at a moment and 3 other forms the next day. You will probably unvoluntarly erase the export from the previous day. Having some kind of timestamp in the generaed filename could help peopole to identify their files.

For instance `glpi-forms-{YYYY-MM-DD}-{crc32}.json` -> `glpi-forms-2024-09-09-a1b2c3d4.json`. The export date is clear for the user, and the crc32 of the contents (or whatever short hash) prevent conflicts.

This is not a blocker for the current PR, it could be done later.

Reply:
Good idea; I'll add it to the backlog so we can merge this faster :)

Reply:
https://github.com/orgs/glpi-project/projects/7/views/1?pane=issue&itemId=79062592